lib/resources/r4/encounter.yaml

```diff
@@ -169,6 +169,13 @@ fields:
     - display: Millennium Encounter Visit Type
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/71
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-71-visit-type
+    - display: v2 Patient Class
```

Comment:
I dont understand where this is coming from....

http://hl7.org/fhir/valueset-encounter-type.html

Reply:
Talked offline with Kathy. The value set is an example value set